internal/clientcache/internal/daemon/server.go

```diff
@@ -29,6 +30,7 @@ import (
 	"github.com/hashicorp/boundary/version"
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/go-secure-stdlib/parseutil"
+	"github.com/mitchellh/go-homedir"
```

Comment:
I think go-homedir may be out of date. There was a proposal later for `os.UserHomeDir` (https://github.com/golang/go/issues/26463) and I am not seeing `cgo` in the implementation (https://cs.opensource.google/go/go/+/refs/tags/go1.23.0:src/os/file.go;l=574) so maybe we can just use that instead?

Reply:
ty!